content/zh/examples/controllers/nginx-deployment.yaml

```diff
@@ -16,6 +16,6 @@ spec:
     spec:
       containers:
       - name: nginx
-        image: nginx:1.15.4
+        image: nginx:1.7.9
```

Comment:
see https://github.com/kubernetes/website/pull/19198/files#diff-dc77252173c201b05594bf5a6a059eceR19

Reply:
nginx:1.15.4 -> nginx:1.7.9?

Reply:
I rebased so it was updated to latest version!